predicators/approaches/active_sampler_learning_approach.py

```diff
@@ -470,7 +470,7 @@ def _learn_nsrt_sampler(self, nsrt_data: _OptionSamplerDataset,
                             nsrt: NSRT) -> Tuple[NSRTSampler, NSRTSampler]:
         # Build targets.
         gamma = CFG.active_sampler_learning_score_gamma
-        num_a_samp = CFG.active_sampler_learning_num_next_option_samples
+        num_a_samp = CFG.q_function_num_lookahead_samples
```

Comment:
hmmm - nit, but i don't like that this has 'q_function' in the name when other active sampler learning methods are going to be using this flag... Can we just keep the flag name as is?